tpot/base.py

```diff
@@ -185,6 +189,13 @@ def __init__(self, generations=100, population_size=100, offspring_size=None,
             A setting of 2 or higher will add a progress bar during the optimization procedure.
         disable_update_check: bool, optional (default: False)
             Flag indicating whether the TPOT version checker should be disabled.
+        periodic_save_path: path string (default: None)
```

Comment:
Please refactor the `periodic_save_path` name to `periodic_checkpoint_path`. I believe that's a more descriptive term for what it's doing.